apps/api/src/registrant/registrant.service.ts

```diff
@@ -94,12 +95,18 @@ export class RegistrantService {
         .process(async item => {
           const domain = process.env.DOMAIN;
           const token = this.jwtService.sign({ email: item.email, id: item.id });
+          const code = submissionsMap.get(item.id);
+          let body = '';
+
+          if (code && domain) {
+            body = this.processTemplateBody(payload.body, { email: item.email, code, domain });
+          }
 
-          const url = domain
+          const unsubUrl = domain
             ? `https://${domain}/unsubscribe?token=${token}`
-            : `http://localhost:3000/unsubscribe?token=${token}`;
+            : `https://ehpr.gov.bc.ca/unsubscribe?token=${token}`;
```

Comment:
Isn't the prod url already contained in a `DOMAIN` field? how does local testing work with this?